packages/devtools_app/lib/src/screens/network/network_screen.dart

```diff
@@ -248,6 +248,12 @@ class _NetworkProfilerControlsState extends State<_NetworkProfilerControls>
             widget.controller.clear();
           },
         ),
+        const SizedBox(width: denseSpacing),
+        ExportButton(
+          minScreenWidthForTextBeforeScaling:
+              _NetworkProfilerControls._includeTextWidth,
+          onPressed: widget.controller.exportAsHarFile,
```

Comment:
add a method in this class, `_exportData` where you call exportAsHarFile and then push a notification showing where the file was downloaded to. 

see this for an example: https://github.com/flutter/devtools/blob/85a45e5fd7cdf5946da19611dd18513bc28b22bb/packages/devtools_app/lib/src/screens/profiler/profiler_screen.dart#L447